src/SonarScanner.MSBuild.Shim/SonarScanner.Wrapper.cs

```diff
@@ -103,12 +103,13 @@ private static string FindScannerExe(ILogger logger)
             {
                 // We unzip the scanner-cli -[version].zip while in the user's machine, upon first usage of the scanner,
                 // for the case where the scanner runs in Linux so that the Linux file permissions are kept.
-                logger.LogInfo($"Unzipping { SonarScannerVersion}.zip");
                 var zipPath = Path.Combine(binFolder, $"sonar-scanner-cli-{SonarScannerVersion}.zip");
+                logger.LogInfo($"Unzipping {SonarScannerVersion}.zip");
                 if (!File.Exists(zipPath))
                 {
                     logger.LogError($"Could not find {zipPath}");
                 }
+                // System.IO.Compression.ZipFile has zipbomb attack protection: https://github.com/dotnet/runtime/issues/15940
```

Comment:
should this be a dogfood post (maybe we can improve the rule?)

Reply:
Yes, based on the link, the rule should not raise an issue for this method.
I will make a post.

Reply:
Email sent regarding this.